ci-operator/step-registry/openstack/conf/creatednsrecords/openstack-conf-creatednsrecords-ref.yaml

```diff
@@ -1,16 +1,27 @@
 ref:
+  # TODO(mandre): move this ref to openstack-provision-dnsrecords
   as: openstack-conf-creatednsrecords
   from: openstack-installer
   commands: openstack-conf-creatednsrecords-commands.sh
   env:
     - name: BASE_DOMAIN
+    - name: CONFIG_TYPE
+      documentation: |
+        The type of config for the environment to deploy.
+
+        * 'minimal' - Configure the install-config with the minimal options to make it work on
+                      our tests, which requires the usage of floating IPs. This config type doesn't
+                      require a proxy server to be deployed.
+        * 'byon' - Configure the install-config to use a pre-created network (BYON) so it wouln't
```

Comment:
nit: `wouln't` -> `won't` or `will not` or whatever.

Reply:
ok I'll address it in a future PR if possible.